src/Analytics/Adapter/ActiveCampaign.php

```diff
@@ -85,29 +381,12 @@ public function createEvent(Event $event): bool
         
         $query = array_filter($query, fn($value) => !is_null($value) && $value !== '');
 
-        $ch = curl_init();
-        curl_setopt($ch, CURLOPT_URL, $this->endpoint);
-        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-        curl_setopt($ch, CURLOPT_POST, 1);
-        curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 5); 
-        curl_setopt($ch, CURLOPT_TIMEOUT, 10);
-        curl_setopt($ch, CURLOPT_HTTPHEADER, [
-            'Content-Type: application/x-www-form-urlencoded'
-        ]);
-        curl_setopt(
-            $ch,
-            CURLOPT_POSTFIELDS,
-            http_build_query($query)
-        );
-
-        $body = curl_exec($ch);
-
-        if (curl_error($ch) !== '') {
+        try {
+            $this->call('POST', 'https://trackcmp.net/event', [], $query);
```

Comment:
To me, this looks like a random domain. Can we add a comment and explain how it refers back to ActiveCampaign? Maybe even reference to their docs.